### PR TITLE
Added email notification option to receive all own messages

### LINKED ISSUE #16855: Receive emails for own posts
It would be great to have the possibility to configure reception of emails of my own posts. Ideally it is configurable on a per user basis.

Background: When someone is keeping track of an issue in his emails, it is very helpful to have his own posts among these emails.

### LINKED ISSUE #20149: Email notification include my own message
### Feature Description

Can email notifications when created or replied to messages include my own creation and replying to messages?
Can choose from user's own settings?

### Screenshots

_No response_

### PR DESCRIPTION
**Pull request by tyroneyeh**
Closes #20149

Sometimes users want to receive email notifications of messages they create or reply to,
Added an option to personal preferences to allow users to choose

### CODE REVIEW

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -1784,6 +1784,7 @@ settings.mirror_sync_in_progress = Mirror synchronization is in progress. Check
 settings.email_notifications.enable = Enable Email Notifications
 settings.email_notifications.onmention = Only Email on Mention
 settings.email_notifications.disable = Disable Email Notifications
+email_notifications.andyourown = And Your Own Email Notifications
```
Comment (Gusted) [inline options/locale/locale_en-US.ini]:
```suggestion
settings.email_notifications.andyourown = And Your Own Email Notifications
```

File: templates/user/settings/account.tmpl [hunk 2]
```diff
@@ -62,6 +62,7 @@
 									<div class="text">{{$.locale.Tr "settings.email_notifications"}}</div>
 									<div class="menu">
 										<div data-value="enabled" class="{{if eq .EmailNotificationsPreference "enabled"}}active selected {{end}}item">{{$.locale.Tr "settings.email_notifications.enable"}}</div>
+										<div data-value="andyourown" class="{{if eq .EmailNotificationsPreference "andyourown"}}active selected {{end}}item">{{$.i18n.Tr "settings.email_notifications.andyourown"}}</div>
```
Comment (Gusted) [inline templates/user/settings/account.tmpl]:
```suggestion
										<div data-value="andyourown" class="{{if eq .EmailNotificationsPreference "andyourown"}}active selected {{end}}item">{{$.locale.Tr "settings.email_notifications.andyourown"}}</div>
```

**Review by Gusted — CHANGES_REQUESTED** [review]
Translation isn't working. Otherwise seems fine.

**Comment by tyroneyeh** [comment]
> Translation isn't working. Otherwise seems fine.

Sorry, I came from the 1.16 merge and didn't notice the difference, thanks

**Review by Gusted — APPROVED** [review]

**Review by Gusted — APPROVED** [review]
LGTM code-wise

**Review by wxiaoguang — APPROVED** [review]

**Comment by wxiaoguang** [comment]
bot

**Comment by tyroneyeh** [comment]
Strange, locale_en-US.ini has been added, why not display the correct text?

**Comment by wxiaoguang** [comment]
The i18n was put to the incorrect place.

The fix:
* https://github.com/go-gitea/gitea/pull/20518

**Pull request merged**